example_versioned_docs/version-latest/16-closures.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Closures
+id: closures
+slug: /closures
+sidebar_label: 16. Closures
+description: Closures with Wing
+keywords: [Wing language, variadic]
+---
+
+Closures are functions that captures variables from its surrounding lexical scope, allowing those variables to persist even after the function is executed outside its original context
```

Comment:
side note - I was tweeting about this the other day, it's kinda surprising how some languages like C and Zig don't support this: https://twitter.com/rybickic/status/1832142453815513162.

I guess it's a bit more of an advanced feature, but it's fortunate since Wing builds of of JavaScript we don't have to solve it from scratch :-)